### DISCUSSION TITLE
wine: how to disable netfilter in a separate profile?

### DISCUSSION THREAD

**Discussion by BlueMax** [netblue30/firejail#5836] [Q&A] [answered]
Hallo,
i've created a `wine.local` with a custom netfilter to block all external traffic by default (no internet in/out).
But sometimes i do need internet access indeed. For this, i want to create a profile that includes the `wine.profile` and additionally revokes the (previously defined) `netfilter` command. And that's the problem. I cannot find a way to revoke the `netfilter` command. I already tried `ignore netfilter` or setting another one but this doesn't work.

So, how can i revoke/ignore the `netfilter` i defined earlier (back to default)?

**Comment by rusty-snake** [comment, marked as answer]
`ignore netfilter` but it has to come before `netfilter`.

You can also use the  command line like 

```
firejail --ignore=netfilter --profile=wine wine foobar
```

Also, does your test look like? Because if your check is connectifity to a domain other commands can block network / name resolution as well. If you want to check if netfilter is used or not, run with `--debug`.

**Comment by BlueMax** [comment]
> it has to come before `netfilter`.

Works, thank you!
It needs an `ignore netfilter` and `ignore net`.
